spec/requests/users_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users controller', type: :request do
+  describe 'GET /index' do
+    before :each do
+      get '/users'
+    end
+    it 'renders template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'correct status' do
+      expect(response.status).to eq(200)
+    end
+    it 'body includes correct text' do
+      expect(response.body).to include('display list of users')
+    end
+  end
+
+  describe 'GET /show' do
+    before :each do
+      get '/users/1'
+    end
+    it 'renders template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'correct status' do
+      expect(response.status).to eq(200)
+    end
+    it 'body includes correct text' do
+      expect(response.body).to include('display certain user list of posts')
+    end
```

Comment:
- [ ] <b>REQUIRED:</b> This test case is failing, Kindly try to fix the issue.
![Screen Shot 2022-03-25 at 02 13 27](https://user-images.githubusercontent.com/45217364/160030067-d30e0dc8-1d76-4ca5-9d12-c7b5a58f69b2.png)